app/javascript/controllers/autocomplete_controller.js

```diff
@@ -0,0 +1,98 @@
+import { Controller } from "@hotwired/stimulus"
+
+// TODO: Add suggest help text and aria-live
```

Comment:
Is this TODO for later (other PR)?

Reply:
Yeah, it's kind of a burden to add this here and requires some extra style and i18n work.